cmake/alpakaCommon.cmake

```diff
@@ -291,280 +291,150 @@ endif()
 #-------------------------------------------------------------------------------
 # Find CUDA.
 if(ALPAKA_ACC_GPU_CUDA_ENABLE)
+    include(CheckLanguage)
+    check_language(CUDA)
 
-    if(NOT DEFINED ALPAKA_CUDA_VERSION)
-        set(ALPAKA_CUDA_VERSION 9.0)
-    endif()
+    if(CMAKE_CUDA_COMPILER)
+        enable_language(CUDA)
+        find_package(CUDAToolkit 9.0 REQUIRED)
 
-    if(ALPAKA_CUDA_VERSION VERSION_LESS 9.0)
-        message(WARNING "CUDA Toolkit < 9.0 is not supported!")
-        set(_ALPAKA_FOUND FALSE)
+        set_target_properties(alpaka PROPERTIES
+                              CUDA_STANDARD ${ALPAKA_CXX_STANDARD}
+                              CUDA_STANDARD_REQUIRED ON) # Don't decay to older standards
```

Comment:
```suggestion
        target_compile_features(alpaka
            PUBLIC cxx_std_${ALPAKA_CXX_STANDARD})
```

Reply:
We are already setting `cxx_std_${ALPAKA_CXX_STANDARD}` in a different place but `cuda_std_${ALPAKA_CXX_STANDARD}` seems to work.